Makefile

```diff
@@ -35,26 +44,48 @@ clean: \
 check-syntax-errors: .
 	@$(call help,$@:)
 	@python -m compileall -q $?
+	@$(call pass,$@:)
+
+## Run linters to check for syntax and style errors.
+check-style: \
+	check-style-all \
```

Comment:
The difference between these two targets is not clear enough from their names, and is actually counterintuitive: `check-style-all` does less than `check-style` 🙃
I don't understand enough what each does to suggest a better name.

Reply:
I tried to avoid changing names, but I agree.

The idea behind is, for all checks (python, types, lint):

* Run the baseline
* Run a subset of more strict rules while I move module to module
* Be sure there is no inconsistency (the all should always be more relaxed that the stricts, but always a superset).

Reply:
Changed, let me know what do you think.